src/main/resources/assets/mythicmetals/lang/de_de.json

```diff
@@ -0,0 +1,1146 @@
+  "abilities.mythicmetals.upgrade_tooltip": "Mythril Bohrer Upgrade",
+  "abilities.mythicmetals.water_protection": "Natürlicher Waterschutz",
+
+    "tooltip.midas_gold.level.0": "Kann in einem Schmiedetisch gefaltet werden, unter Verwendung von Midas Gold Blöcken",
+  "tooltip.midas_gold.level.1": "Man braucht eine ganze Menge, um seine wahre Macht zu entfalten...",
+  "tooltip.midas_gold.level.2": "Über hundert Blöcke. Beeindruckend! Mach weiter!",
+  "tooltip.midas_gold.level.3": "Warum solltest du so viele Blöcke machen? Mach einfach normales Gold...",
+  "tooltip.midas_gold.level.4": "Das Schwert fühlt sich mächtiger an... Kann es weiter gehen?",
+  "tooltip.midas_gold.level.5": "König Midas wäre stolz auf diese Waffe",
+  "tooltip.midas_gold.level.6": "König Midas wäre stolz auf diese Waffe",
+  "tooltip.midas_gold.level.7": "König Midas wäre stolz auf diese Waffe",
+  "tooltip.midas_gold.level.8": "König Midas wäre stolz auf diese Waffe",
+  "tooltip.midas_gold.level.9": "König Midas wäre stolz auf diese Waffe",
+  "tooltip.midas_gold.level.10": "Eine großartige Waffe! Vielleicht gibt es eine Möglichkeit, noch weiter zu gehen...",
+  "tooltip.midas_gold.level.11": "Ein königliches Upgrade für deine Klinge",
+  "tooltip.midas_gold.level.12": "Wenn du beim ersten Mal keinen Erfolg hast...",
+  "tooltip.midas_gold.level.13": "... Versuche es noch einmal. Dann sollte dein Mut erscheinen.",
+  "tooltip.midas_gold.level.14": "Die wahre Macht des Königlichen Midas-Gold-Schwerts wird sich erst zeigen...",
+  "tooltip.midas_gold.level.15": "... wenn du das Ende dieser Reise erreicht hast.",
+  "tooltip.midas_gold.level.16": "Durch tausendmaliges Falten wird die Klinge des Schwertes rein.",
+  "tooltip.midas_gold.level.17": "Dass alles, was du besiegst, dir mehr von dem gibt, was du wirklich willst:",
+  "tooltip.midas_gold.level.18": "Noch mehr Midas-Gold!",
+  "tooltip.midas_gold.level.19": "Die letzte Etappe liegt vor dir, du kannst es schaffen!",
+  "tooltip.midas_gold.level.20": "Die unglaubliche Macht von König Midas ruht jetzt in deiner Klinge!",
+  "tooltip.midas_gold.level.21": "Kurty00 war hier",
+  "tooltip.midas_gold.level.23": "Die unglaubliche Macht von König Midas ruht nun in Deiner Klinge ...?",
+  "tooltip.midas_gold.level.24": "Wendet Ihr immer noch Midas-Gold auf dieses Schwert an?",
+  "tooltip.midas_gold.level.25": "Warum wendest du immer noch Midas-Gold auf dieses Schwert an?",
+  "tooltip.midas_gold.level.26": "Du hast bereits den maximalen Schaden erreicht, du hast die Spezialfähigkeit",
+  "tooltip.midas_gold.level.27": "Irgendwann erreichst du einfach ein Cap oder so",
+  "tooltip.midas_gold.level.28": "Oder vielleicht geht das ewig so weiter... Ich weiß nicht mehr, ob es so etwas gibt",
+  "tooltip.midas_gold.level.29": "Du hast vielleicht einen Schritt zurück gemacht...",
+  "tooltip.midas_gold.level.30": "Wir reden nicht über Stufe 21...",
+  "tooltip.midas_gold.level.31": "Platzhaltertext",
```

Comment:
This line should intentionally be left untranslated, as the joke is the manner that this line seems unfinished.